.gitignore

```diff
@@ -170,3 +170,6 @@ zz_generated.openapi.go
 
 # sshuttle
 sshuttle.pid
+
+# manifests
```

Comment:
That's an error in the generation nothing which we should add to gitignore..

Reply:
I  think it happens when the repo is in the GOPATH but not under $GOPATH/src/sigs.k8s.io/cluster-api-provider-openstack

xref: https://github.com/kubernetes-sigs/cluster-api-provider-openstack/issues/801

Reply:
@jichenjc I assumed this happened with the current master.

How is your setup regarding GOPATH and folder location?

Reply:
yes, master, 
GOPATH is not set, I only run `make generate` at my folder 

~/go/src/sigs.k8s.io/cluster-api-provider-openstack/

previously there is no such issue and after some changes recently I noticed such file exist

Reply:
Okay thx. I"ll try to fix it

Reply:
This might fix it: https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/827